modules/ding_facetbrowser/plugins/content_types/ding_facetbrowser/ding_facetbrowser.inc

```diff
@@ -30,8 +30,8 @@ function ding_facetbrowser_content_type_render($subtype, $conf, $panel_args, $co
   if (is_string($executed_search_module)) {
     $results = module_invoke($executed_search_module, 'ding_facetbrowser');
     if (is_object($results)) {
-      foreach ($results->facets as $result) {
-        if (count($result->terms) == TRUE) {
+      foreach ($results->facets as $facet) {
+        if (count($facet->getTerms()) > 0) {
```

Comment:
The class `TingSearchFacetTerm` does.

In this regard a `TingSearchFacet` object contains an array of `TingSearchFacet` objects which has a cout for how many times the term occurs in the search result.

Example: A search result for "Harry Potter" may contain the facet "Material type". This facet may contain the term "Book". Book may have the count 42 because it occurs 42 times within the search result.

Reply:
👍